examples/codegen/data_types.rs

```diff
@@ -77,7 +78,7 @@ pub struct FooMessage<'a> {
     pub f_sint32: Option<i32>,
     pub f_sint64: i64,
     pub f_bool: bool,
-    pub f_FooEnum: Option<FooEnum>,
+    pub f_FooEnum: Option<data_types::FooEnum>,
```

Comment:
And this is EXACTLY why such files needed a `use self::*` so they could refer to items within the current module. Very hacky! I considered this and found it surprisingly tricky, although worth revisiting. The problem is that at the point of expansion, we don't know the package of the message containing the field.